src/picongpu/include/simulationControl/MovingWindow.hpp

```diff
@@ -220,8 +220,8 @@ class MovingWindow
         const SubGrid<simDim>& subGrid = Environment<simDim>::get().SubGrid();
         Window window;
 
-        window.localDimensions = Selection<simDim>(subGrid.getLocalDomain().size);
-        window.globalDimensions = Selection<simDim>(subGrid.getGlobalDomain().size);
+        window.localDimensions = Selection<simDim>(subGrid.getLocalDomain());
+        window.globalDimensions = Selection<simDim>(subGrid.getGlobalDomain());
 
         /* If sliding is inactive, moving window is the same as global domain (substract 0)*/
```

Comment:
new comment: `/* the moving window is smaller than the global domain by *one GPU* in moving (y) direction */`